src/phoebus_boundaries/phoebus_boundaries.cpp

```diff
@@ -32,6 +33,8 @@ using namespace parthenon::package::prelude;
 
 namespace Boundaries {
 
+parthenon::TopologicalElement CC = parthenon::TopologicalElement::CC;
```

Comment:
This code could probably be cleaned up now that @lroberts36 exposed generic boundary conditions but I went for the minimalist approach of just tweaking what we have